.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -57,3 +57,12 @@ Tested on:
 ### Additional Information
 
 <!-- Anything else we should know? -->
+
+## Summary
+### Behavior Before Change
```

Comment:
I think this should go in the proposed changes section instead, to avoid duplication.

Reply:
Done